cli/src/templates/base/package.json.ejs

```diff
@@ -3,69 +3,50 @@
   "version": "1.0.0",
   "scripts": {
     "android": "expo start --android",
-	<% if (props.packageManager === "bun") { %>
```

Comment:
@falcoagustin can you include these changes in your PR? @ernestoresende when i was testing, I think this change was necessary while using bun